api/src/main/java/org/openmrs/module/fgh/mpi/MpiIntegrationProcessor.java

```diff
@@ -1,24 +1,20 @@
 package org.openmrs.module.fgh.mpi;
 
-import static java.lang.Boolean.valueOf;
-import static org.openmrs.module.debezium.DatabaseOperation.CREATE;
-import static org.openmrs.module.debezium.DatabaseOperation.DELETE;
-import static org.openmrs.module.fgh.mpi.MpiConstants.FIELD_ACTIVE;
-import static org.openmrs.module.fgh.mpi.MpiConstants.FIELD_CONTACT;
-import static org.openmrs.module.fgh.mpi.MpiConstants.FIELD_ID;
-import static org.openmrs.module.fgh.mpi.MpiConstants.FIELD_RELATIONSHIP;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.openmrs.module.debezium.DatabaseEvent;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.Boolean.valueOf;
+import static org.openmrs.module.debezium.DatabaseOperation.CREATE;
+import static org.openmrs.module.debezium.DatabaseOperation.DELETE;
+import static org.openmrs.module.fgh.mpi.MpiConstants.*;
```

Comment:
Don't use these imports

Reply:
Please make sure you use the IDE to organize the imports using the OpenMRS import order where static imports come first

Reply:
Infact just revert all changes in this file since it had no real changes except these import changes.